### DISCUSSION TITLE
Aggregation button disabled

### DISCUSSION THREAD

**Discussion by jochenberger** [eclipse-birt/birt#855] [General]
Hi,

we tried to upgrade to 4.9. The results look very promising (:clinking_glasses:), but it looks as if there is still an issue with the aggregation button. See https://bugs.eclipse.org/bugs/show_bug.cgi?id=511212 for reference.

Cheers,
Jochen

**Comment by claesrosell** [comment]
I looked into this bug a couple of months ago, and I found the cause for it. If I recall correctly it was a refactoring done when introduction of new functionality that caused this to brake. However, my knowledge about the birt designer was is too limited to understand how the obvious fix were to effect other parts of the designer.

If you file a new issue I will take another look at it. I will probably have some questions along the way though.

**Comment by wimjongman** [comment]
Issue solved by https://github.com/eclipse/birt/issues/856
